vms/platformvm/warp/validator_test.go

```diff
@@ -304,3 +305,48 @@ func TestSumWeight(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkGetCanonicalValidatorSet(b *testing.B) {
+	pChainHeight := uint64(1)
+	subnetID := ids.GenerateTestID()
+	numNodes := 10_000
+	getValidatorOutputs := make([]*validators.GetValidatorOutput, 0, numNodes)
+	for i := 0; i < numNodes; i++ {
+		nodeID := ids.GenerateTestNodeID()
+		blsPrivateKey, err := bls.NewSecretKey()
+		if err != nil {
+			b.Fatal(err)
+		}
+		blsPublicKey := bls.PublicFromSecretKey(blsPrivateKey)
+		getValidatorOutputs = append(getValidatorOutputs, &validators.GetValidatorOutput{
+			NodeID:    nodeID,
+			PublicKey: blsPublicKey,
+			Weight:    20,
+		})
+	}
+
+	getNValidatorsMap := func(n int) map[ids.NodeID]*validators.GetValidatorOutput {
+		validators := make(map[ids.NodeID]*validators.GetValidatorOutput)
+		for i := 0; i < n; i++ {
+			validator := getValidatorOutputs[i]
+			validators[validator.NodeID] = validator
+		}
+		return validators
+	}
+
+	for _, size := range []int{0, 1, 10, 100, 1_000, 10_000} {
+		validatorsOutput := getNValidatorsMap(size)
+		validatorState := &validators.TestState{
+			GetValidatorSetF: func(ctx context.Context, height uint64, subnetID ids.ID) (map[ids.NodeID]*validators.GetValidatorOutput, error) {
```

Comment:
These args aren't used
```suggestion
			GetValidatorSetF: func(context.Context, uint64, ids.ID) (map[ids.NodeID]*validators.GetValidatorOutput, error) {
```